packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -1450,6 +1450,54 @@ const schema: SchemaType<DbUser> = {
     label: "Opted-in to receiving invitations for dialogue facilitation from LessWrong team",
```

Comment:
While you're here, can you put this into a form group so it's not hanging out at the top up here?  (Or just put it into a separate PR right now)

![Uploading image.png…]()